DOL.WHD.Section14c.Web/src/modules/components/accountManagementControls/accountFormController.js

```diff
@@ -101,7 +102,29 @@ module.exports = function(ngModule) {
     };
 
     vm.cancelClick = function() {
-      $location.path('/');
+      $location.path('/admin/users');
     };
+
+    vm.update = {
+      status: "Inprogress",
+      message: 'In Progress ...'
+    };
+    vm.updateUserAccountModalIsVisible = false;
```

Comment:
This made me laugh a little.  Thank y'all for being patient and responsive over this whole engagement.